_posts/2018-09-28-3D-Printing-Intro.md

```diff
@@ -0,0 +1,400 @@
+---
+layout: post
+authors: [kevin_van_den_abeele]
+title: "3D Printing: An introduction"
+image: /img/2018-09-28-3D-Printing/banner.jpg
+tags: [3D printing, printing, building, technology, iot, internet of things, SLA, DLP, FDM, SLS, PLA, ABS, PETG, printer, sintering, melting, construction, tinkering]
+category: IoT
+comments: true
+---
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/css/lightbox.css" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/bootstrap.css" />
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/js/lightbox.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/index.min.js"></script>
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [What is 3D printing](#what-is-3d-printing)
+3. [Types of 3D printing](#types-of-3d-printing)
+4. [Getting started with 3D printing](#getting-started-with-3d-printing)
+5. [The future](#the-future)
+
+## Introduction
+
+3D printing is a term that has been hyped for a long time.
+It's a technology that is in essence not that new, but now more than ever is getting better and accessible to everyone.
+Today we take a dive into the world of 3D printing and what is really has in store for us and the worlds in the coming years.
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/sla-glow.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 500px;">
+
+
+## What is 3D printing
+
+Basically 3D printing can be described as: Producing 3D modeled objects by printing them with a 3D printer.
+It offers a new way to do fast prototyping without the need to create very expensive molds or stencils.
+
+It has long been hyped to be the next big thing:
+
+- Everyday use, recreating objects, replacing broken parts
+- Medical use, create patient specific casts or prostheses that fit better
+- Weapons (in the news), 3D print a gun, can pass through metal detector
```

Comment:
maybe add a link to in the news
print a gun, can pass => print a gun, which can pass